README.rst

```diff
@@ -32,20 +32,21 @@ Scene Select Module creation
 ----------------------------
 Modules are built off the module-prod branch. Create an annotated tag to tag a module build.
 
-e.g.
+1. make sure the latest changes in the master branch are brought/sync-ed to the module-prod branch. To do this, create a PR from the master to the module-prod branch
+2. once the PR is approved, merge it to the module-prod 
+3. login or sudo as lpgs in a terminal. This is because production modules can must be built as the lpgs user
+4. head to the path, "/home/547/lpgs/sandbox/dea-ard-scene-select/module". Run "cd /home/547/lpgs/sandbox/dea-ard-scene-select/module/"
+5. checkout the "module-prod" branch. Run "git checkout module-prod"
+6. tag the new version. Here's what I ran for the most recent build:
```

Comment:
```suggestion
6. tag the new version. This does not have to be done as lpgs. For example:
```
This does not have to be done as lpgs - that's my understanding anyhow. But tag's haven't been working for me.